lib/uo_cli.rb

```diff
@@ -372,9 +372,6 @@ def opt_des_params
           opt :feature, "\nPath to the feature JSON file\n" \
             "Example: uo des_params --sys-param-file path/to/sys_params.json --feature path/to/example_project.json\n", type: String, required: true, short: :f
 
-          opt :model_type, "\nSelection for which kind of DES simulation to perform\n" \
-            "Valid choices: 'time_series'", type: String, default: 'time_series'
```

Comment:
AAAAAAAArgh! @tanushree04 You were right to not include it from the beginning! I apologize! I'll make a PR removing `model_type` (because it's incompatible with what the GMT now expects).